src/Response/Directives/Display/Image.php

```diff
@@ -16,4 +16,37 @@ class Image
      * @var ImageSource[]
      */
     public $sources = [];
+
+    /**
+     * @param ImageSource $imageSource
+     */
+    public function addImageSource(ImageSource $imageSource)
+    {
+        $this->sources[] = $imageSource;
+    }
+
+    /**
+     * @param string|null $contentDescription
+     * @param null|ImageSource|ImageSource[] $imageSource
```

Comment:
This should be `@param ImageSource[] $imageSource` because a param should be array oder Object[|null] but not both.